modules/ding_dibs/ding_dibs.module

```diff
@@ -211,6 +211,10 @@ function ding_dibs_capture($delta = NULL, array &$transaction = NULL) {
   $param['orderid'] = $transaction['payment_order_id'];
   $param['transact'] = $transaction['payment_transaction_id'];
   $param['textreply'] = TRUE;
+  // If this parameter is present, the parameter orderid has to be unique
+  // compared to all other orderid's used by the merchant.
+  // If the orderid isn't unique, the call will be declined by a reason=7.
+  $param['uniqueoid'] = 'yes';
```

Comment:
Could it ever be possible that an orderid might show up on upgrade-fps/testing environments?
Because if that can happen, then a payment would possibly be rejected on production with this change